### PR TITLE
Integrated Mayhem

### PR DESCRIPTION
**Pull request by bmwtsn098**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM golang:1.18
```
Comment (xansec) [inline Dockerfile:1]: Please use package stage.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Namespaced project name that the target belongs to
```
Comment (xansec) [inline Mayhemfile:1]: Please omit image name and use generic project name.

**Comment by bmwtsn098** [comment]
I made the requested changes. Let me know if everything looks good. Thanks

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**